src/main/java/com/booksaw/betterTeams/text/Legacy.java

```diff
@@ -0,0 +1,117 @@
+package com.booksaw.betterTeams.text;
```

Comment:
Literally the only thing I can find that needs changing on this PR, could this class be renamed to be more specific, Legacy is a little broad.